ViewGenerator/tools/webpack/Rules/Files.ts

```diff
@@ -38,7 +42,11 @@ const getResourcesRuleSet = (assemblyName?: string, pluginsBase? : string): Rule
                                 }
 
                                 // URL (argument) is a relative path and contains the resource base path or the plugin assembly in its content
-                                return url.substring(idx);
+                                if(isWebComponentsPackage) {
+                                    return "../node_modules/webview.plugins/@outsystems/web-" + url.substring(url.indexOf("/packages/") + "/packages/".length);
```

Comment:
```
let package = url.substring(url.indexOf(PackagesFolder) + PackagesFolder.length);
return `/${pluginsBase}/node_modules/@outsystems/web-${package}`;
```

Reply:
pluginsBase corresponds to ServiceStudio.WebViewImplementation.Framework. It should be always filled when building a project with views